docs/products/opensearch/reference/advanced-params.rst

```diff
@@ -0,0 +1,6 @@
+List of advanced parameters
+============================
+
+When creating or working with an Aiven for OpenSearch service, several configuration options are available. They are summarized below:
```

Comment:
Several usually means "more than two but not many.", in our case it feels like way more than several :D Let's rephrase it a bit.

For "When creating or working" would it be more concise to use "When working with". It summarises all scenarios and is shorter